src/ssl.c

```diff
@@ -11683,6 +11708,29 @@ int wolfSSL_set_cipher_list(WOLFSSL* ssl, const char* list)
 #endif
 }
 
+
+int wolfSSL_set_cipher_list_bytes(WOLFSSL* ssl, const byte* list,
+                                  const int listSz)
+{
+    WOLFSSL_ENTER("wolfSSL_set_cipher_list_bytes");
+#ifdef SINGLE_THREADED
+    if (ssl->ctx->suites == ssl->suites) {
+        ssl->suites = (Suites*)XMALLOC(sizeof(Suites), ssl->heap,
+                                       DYNAMIC_TYPE_SUITES);
+        if (ssl->suites == NULL) {
+            WOLFSSL_MSG("Suites Memory error");
+            return MEMORY_E;
+        }
+        *ssl->suites = *ssl->ctx->suites;
```

Comment:
Please replace this with an `XMEMCPY` -> `XMEMCPY(ssl->suites, ssl->ctx->suites, sizeof(Suites));`.